ci-operator/step-registry/insights-qe/iqe-cost-management-plugin/insights-qe-iqe-cost-management-plugin-commands.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "Run Cost Management Interop testing..."
+SECRETS_DIR="/tmp/secrets/ci"
+
+CONSOLE_URL=$(cat $SHARED_DIR/console.url)
+OCP_API_URL="https://api.${CONSOLE_URL#"https://console-openshift-console.apps."}:6443"
+OCP_CRED_USR="kubeadmin"
+OCP_CRED_PSW="$(cat ${SHARED_DIR}/kubeadmin-password)"
+oc login ${OCP_API_URL} --username=${OCP_CRED_USR} --password=${OCP_CRED_PSW} --insecure-skip-tls-verify=true
+
+# create secret for cluster login
+oc create secret generic kubeconfig-secret --from-file $SHARED_DIR/kubeconfig
+
+# create secret for pulling quay image
+username="$(cat ${SECRETS_DIR}/username)"
+password="$(cat ${SECRETS_DIR}/password)"
+oc create secret docker-registry cspi-pull-secret --docker-server=quay.io \
+--docker-username=${username} --docker-password=${password}
+
+# create secret for vault access
+oc apply -f $SECRETS_DIR/insights-vault-pull
```

Comment:
The above secret are used for pulling quay images, I am not sure why this secret is needed. What does this vault contain?

Reply:
@Simranpal insights-vault-pull is the credential to access vault.devshift.net. It includes url, role_id and secret_id.